src/emulator/config.cpp

```diff
@@ -135,16 +136,17 @@ static bool deserialize(Config &cfg) {
     get_yaml_value(config_node, "log-uniforms", &cfg.log_uniforms, false);
     get_yaml_value(config_node, "pstv-mode", &cfg.pstv_mode, false);
     get_yaml_value(config_node, "show-gui", &cfg.show_gui, false);
+    get_yaml_value(config_node, "show-game-background", &cfg.show_game_background, true);
     get_yaml_value(config_node, "icon-size", &cfg.icon_size, static_cast<int>(64));
     get_yaml_value(config_node, "archive-log", &cfg.archive_log, false);
     get_yaml_value(config_node, "texture-cache", &cfg.texture_cache, true);
     get_yaml_value(config_node, "sys-button", &cfg.sys_button, static_cast<int>(SCE_SYSTEM_PARAM_ENTER_BUTTON_CROSS));
     get_yaml_value(config_node, "sys-lang", &cfg.sys_lang, static_cast<int>(SCE_SYSTEM_PARAM_LANG_ENGLISH_US));
+    get_yaml_value(config_node, "background-image", &cfg.background_image, static_cast<std::string>(""));
     get_yaml_value(config_node, "background-alpha", &cfg.background_alpha, 0.000f);
-    get_yaml_value_optional(config_node, "log-level", &cfg.log_level, static_cast<int>(spdlog::level::trace));
-    get_yaml_value_optional(config_node, "pref-path", &cfg.pref_path);
+    get_yaml_value(config_node, "log-level", &cfg.log_level, static_cast<int>(spdlog::level::trace));
+    get_yaml_value(config_node, "pref-path", &cfg.pref_path, static_cast<std::string>(""));
```

Comment:
The fact that this work is amazing

```suggestion
    get_yaml_value(config_node, "pref-path", &cfg.pref_path, std::string{});
```